app/models/course/enrol_request.rb

```diff
@@ -0,0 +1,27 @@
+class Course::EnrolRequest < ActiveRecord::Base
+  stampable
+  has_paper_trail on: [:destroy]
+  validates_uniqueness_of :course_id, scope: :user_id
+
+  scope :student, -> { where(role: CourseUser.roles[:student]) }
+  scope :staff, -> { where(role: [CourseUser.roles[:teaching_assistant],
```

Comment:
Use the `lambda` method for multi-line lambdas.<br>Block body expression is on the same line as the block start.